src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +91,12 @@ public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(s -> s.getAge() >= femaleAge && s.getSex() == Person.Sex.WOMAN)
+                .map(s -> s.getCats())
+                .flatMap(s -> s.stream())
```

Comment:
You can do it in one line
```suggestion
                .flatMap(person -> person.getCats().stream())
```